unit-tests/C/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+#  minimum required cmake version: 3.1.0
+cmake_minimum_required(VERSION 3.1.0)
+
+project(unit-tests-rsutil-c)
```

Comment:
I would have expected:
```
project(unit-tests-rsutil-c C)
```
To make is a C compile...
How is this a C compilation otherwise?